packages/@vuepress/shared/src/utils/resolveHeadIdentifier.ts

```diff
@@ -8,8 +8,8 @@ export const resolveHeadIdentifier = ([
   attrs,
   content,
 ]: HeadConfig): string => {
-  // avoid duplicated `<meta>` with same `name`
-  if (tag === 'meta' && attrs.name) {
+  // avoid duplicated `<meta>` with same `name` except the name `theme-color`
+  if (tag === 'meta' && attrs.name && attrs.name !== 'theme-color') {
```

Comment:
I'd say this is another issue, and we'd better not change this in current PR.

Reply:
Okey, reverted in https://github.com/vuepress/vuepress-next/pull/218/commits/bf81653b01df39100bbd74f5c725be402c3a2ca8